src/daemon/ca/certauth.rs

```diff
@@ -1145,24 +1177,21 @@ impl CertAuth {
     /// the pending key appropriately, finally it will also return a
     /// publication event for the matching key if publication is needed.
     ///
-    /// In future, when ROAs and delegating certificates are supported, this
-    /// should be updated to also generate appropriate events for changes
-    /// affecting these objects if needed - e.g. because resources were lost
-    /// and ROAs/Certs would be become invalid.
+    /// This will also generate appropriate events for changes affecting
```

Comment:
Does this correction of the comment reflect that the behavioural change is introduced by this PR, or was the comment unrelated to this PR but out-dated and has been brought up-to-date here?